server/src/collection.rs

```diff
@@ -205,6 +205,34 @@ impl Collection {
         None
     }
 
+    #[allow(unused)]
```

Comment:
what is this for?

Reply:
This is to avoid the unused clippy warning.